x-pack/legacy/plugins/canvas/public/components/color_manager/color_manager.tsx

```diff
@@ -74,7 +74,7 @@ export const ColorManager: FunctionComponent<Props> = ({
         <EuiFieldText
           value={value}
           isInvalid={!validColor && value.length > 0}
-          placeholder={strings.getCodePlaceholder()}
+          placeholder={'strings.getCodePlaceholder()'}
```

Comment:
What is this??

Reply:
Oops, good catch! I had it disabled temporarily while I was figuring out what was crashing Storybook in this component. And then forgot to turn it back. Now it's all good, reverted.